app/nginx/nginx.conf

```diff
@@ -0,0 +1,9 @@
+
+server {
+    listen       80;
+    location / {
+        root   /usr/share/nginx/html;
+        index  index.html index.htm;
+        try_files $uri $uri/ /index.html;                 
+    }
+}
```

Comment:
so this would serve the react app on port 80?

Reply:
yep